wp-content/themes/pub/wporg-learn-2020/archive-course.php

```diff
@@ -16,6 +16,8 @@
 get_header(); ?>
 
 <main id="main" class="site-main">
+	<?php get_template_part( 'template-parts/component', 'breadcrumbs' ); ?>
```

Comment:
I like that you made a separate/atomic commit for updating these files, so that the the history is clean 👍🏻